static/pages/partnerships/Partnerships.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+oppiaFoundationWebsite.controller(
+  'PartnershipsPage', ['$scope', '$http', function($scope, $http) {
+    $scope.submitContactUsForm = function(
+        fullName, email, organization, comment) {
+      var _MAILHANDLER_URL = '/ajax/mailhandler';
+      $http.post(_MAILHANDLER_URL, {
+        name: fullName,
```

Comment:
In app.yaml, put your ajax handler above the first one. The handlers are called in order and the static file handler is shadowing the backend python one, because its URL regex matches.

Reply:
Wow, you are right on the money, Sean. Thanks a bunch!